common/models/logbook.js

```diff
@@ -214,10 +218,18 @@ async function getUserProposalIds(userId) {
       roleNameList.push(user.username);
       options.currentGroups = roleNameList;
     }
+    options.currentGroups = [...new Set(options.currentGroups)];
 
-    const proposals = await Proposal.find({
-      where: { ownerGroup: { inq: options.currentGroups } },
-    });
+    let proposals = {};
+    if ( options.currentGroups.includes("admin") ) {
```

Comment:
isn't this managed already by the onwable model: https://github.com/SciCatProject/backend/blob/master/common/models/ownable.js#L16-L43?

Reply:
I do not think so, at least, I did not find this feature.
Before my change, access to the logbook was still managed locally. I juts extended the logic to include access groups.

Reply:
I thought that the find of every method which inherits from ownable calls the before("access") operation hook, thus the lines that I've shared. The check there would be on global access though rather than admin